app/js/tests/test_form_pages.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+var test = require( 'tape' ),
+	formPages = require( '../lib/form_pages' ),
+	Promise = require( 'promise' ),
+	pageSpy = {
+		display: false,
```

Comment:
Just a minor thing: I find this variable name misleading and personally would prefer something like `displayed` or `visible`. Will merge this now, though.